Gemfile

```diff
@@ -10,20 +10,25 @@ group :development do
   gem 'pry'
   gem 'pry-stack_explorer', platforms: %i[ruby_19 ruby_20 ruby_21]
   gem 'rubocop', '0.52.1'
+
+  group :test do
+    # ActiveSupport required to test compatibility with ActiveSupport Core Extensions.
+    # rubocop:disable Bundler/DuplicatedGem
+    require File.expand_path('../lib/hashie/extensions/ruby_version', __FILE__)
+    if Hashie::Extensions::RubyVersion.new(RUBY_VERSION) >=
+       Hashie::Extensions::RubyVersion.new('2.4.0')
+      gem 'activesupport', '~> 5.x', require: false
+    else
+      gem 'activesupport', '~> 4.x', require: false
+    end
+    # rubocop:enable Bundler/DuplicatedGem
+    gem 'rake'
+    gem 'rspec', '~> 3'
+    gem 'rspec-pending_for', '~> 0.1'
+  end
```

Comment:
are nested groups like this a common thing?

Reply:
I've never seen this. Usually it's `group :development, :test`, then another one `group :test`. I think this is actually better because you don't have to chase both blocks. I don't care enough tho 🚀

Reply:
I use them all the time for things like this. It allows you to have fewer blocks, which means fewer contexts, which means it's easier to understand.